pkg/services/accesscontrol/roles.go

```diff
@@ -3,8 +3,9 @@ package accesscontrol
 import "github.com/grafana/grafana/pkg/models"
 
 var ldapAdminReadRole = RoleDTO{
-	Name:    ldapAdminRead,
-	Version: 1,
+	Name:        ldapAdminRead,
+	Version:     2,
+	Description: "Allows to read LDAP information and status.",
```

Comment:
```suggestion
	Description: "Allows to read LDAP and status information.",
```
Allows [what? / you?]

Reply:
Thanks @osg-grafana for reviewing. I'm no expert here, but I'll try to be useful :slightly_smiling_face: 

Side note: A role is a list of permissions that can be granted to: User(s), Team(s), Builtin Role(s) ("Viewer", "Editor", "Admin", "Grafana Admin"). In the end, thanks to an inheritance mechanism, it always allow an end user to perform an "action" (often on a "scope").
Here we have two actions from what I see, LDAPUsersRead (which most probably protects the http endpoint to list ldap users) and ActionLDAPStatusRead (which most probably protects the http endpoint to get the connection status of Grafana with the LDAP server(s))

With that in mind, do you think this description is clearer:
```suggestion
	Description: "Role to allow users to read LDAP information and get LDAP connection status.",
```
However, since it's a role description,  `"Role to allow users to"` seems a bit redundant, I think that's why the description is currently `"Allows to..."`

Reply:
I'd remove the first part of the sentence entirely.  We don't want to say who is doing it, because we may be adding more in the future (e.g. service accounts).  Unless you want something like "Allows client" or "Allows supplicant"

We can put something in the column header like "These permissions allow a client to..."